app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  has_many :posts, foreign_key: 'author_id'
```

Comment:
- [ ] Just as you have done with posts, the user has many comments and likes, using the `author_id` as foreign key. In Rails, it is important to specify this since the column name is different from the model name.